src/components/SearchMap.js

```diff
@@ -36,8 +36,8 @@ const SearchMap = ({mapLoaded, region, updateRegion, sites}) => {
             return (
                 <MapView
                     style={fillScreen}
-                    region={region}
-                    onRegionChange={onRegionChange}
+                    initialRegion={lastKnownRegion}
+                    onRegionChangeComplete={onRegionChange}
```

Comment:
Main change for the fix in GH-20; we're no longer forcing the `region` from state. Instead of having the `MapView` take its `region` from state, we're letting it scroll freely, and instead of firing off every `onRegionChange` event, we're going to try using `onRegionChangeComplete` to track the `lastKnownRegion`. Hopefully the `onRegionChangeComplete` native debouncing will also work for us out-of-the-box when we add in search functionality.